### DISCUSSION TITLE
Sould Demolition make a new release now ?

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#45] [Polls]
a

**Comment by ThatOneSillyGuyDon** [comment]
bro, we still need to fix the softcoded menu shit before we can release it

**Reply by RealDealJinx** [reply]
im sowwry demowy wemowy :( /j

**Reply by ThatOneSillyGuyDon** [reply]
> im sowwry demowy wemowy :( /j

YOU MADE ME ANGY MY DISCORD KITTEN >:( /j

It’s fine, we probably just need to re-add the other 2 menu buttons

**Reply by RealDealJinx** [reply]
> > im sowwry demowy wemowy :( /j

>YOU MADE ME ANGY MY DISCORD KITTEN >:( /j

>It’s fine, we probably just need to re-add the other 2 menu buttons

ig
